internal/ingress/controller/store/endpoint_test.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
Here's probably 2021.

Will review the tests later, will release the CI to make its part :)